startupscript/butane/aws/set-metadata.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+# Sets tags on the EC2 instance with the given key and value.
+function set_metadata() {
+  echo "Creating tag vwbapp:$1"
+  readonly AWS_CLI_EXE="docker run --rm public.ecr.aws/aws-cli/aws-cli"
+  local id
+  id="$(wget -q -O - http://169.254.169.254/latest/meta-data/instance-id)"
+  ${AWS_CLI_EXE} ec2 create-tags \
```

Comment:
Don't feel strongly about this, but I think this simplifies while still be readable:

```suggestion
  docker run --rm public.ecr.aws/aws-cli/aws-cli \
    ec2 create-tags \
      --resources "${id}" \
      --tags Key=vwbapp:"$1",Value="$2"
```